rest-assured-tests/build.gradle

```diff
@@ -0,0 +1,18 @@
+plugins {
+    id 'java'
+}
+
+group 'io.rest-assured'
+version '1.0-SNAPSHOT'
+
+sourceCompatibility = 1.8
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testCompile 'io.rest-assured:rest-assured:4.1.0'
+    testCompile group: 'junit', name: 'junit', version: '4.12'
```

Comment:
use the same style of dependency declaration. we use the same as you used with restassured dependency

Reply:
fixed